docs/QUICKSTART.md

```diff
@@ -15,7 +15,7 @@ how TUF works, however.  It will serve as a very basic update system.
 
 Make sure that TUF is installed, along with some of the optional cryptographic
 libraries and C extensions.  Try this command to do that:
-`pip install securesystemslib[crypto,pynacl] tuf`
+`pip install colorama securesystemslib[crypto,pynacl] tuf`
```

Comment:
Cool stuff! This could also be:
```suggestion
`pip install securesystemslib[colors,crypto,pynacl] tuf`
```
Just saying. I'm fine either way. :)

Reply:
I prefer that, patch updated. Thanks!